src/modules/map/components/LocationPopup.vue

```diff
@@ -153,86 +116,81 @@ async function updateQrCode(url) {
+        <div data-cy="import-file-content">
+            <ul class="nav nav-tabs nav-justified" role="tablist">
+                <li class="nav-item" role="presentation">
+                    <button
+                        class="nav-link py-1"
+                        :class="{
+                            active: selectedTab === 'position',
+                        }"
+                        data-cy="location-popup-position-tab-button"
+                        type="button"
+                        role="tab"
+                        aria-controls="nav-position"
+                        :aria-selected="selectedTab === 'position'"
+                        @click="
+                            (selectedTab = 'position'),
+                                (showEmbedSharing = false),
+                                (newClickInfo = false)
+                        "
+                    >
+                        {{ $t('position') }}
+                    </button>
+                </li>
+                <li class="nav-item" role="presentation">
+                    <button
+                        ref="copyButton"
+                        class="nav-link py-1 px-0"
+                        :class="{
+                            active: selectedTab === 'share',
+                        }"
+                        data-cy="location-popup-share-tab-button"
+                        type="button"
+                        role="tab"
+                        aria-controls="nav-share"
+                        :aria-selected="selectedTab === 'share'"
+                        @click="requestCopy(), (selectedTab = 'share')"
+                    >
+                        {{ $t('link_bowl_crosshair') }} &nbsp;&nbsp;<FontAwesomeIcon
+                            v-if="currentLang != 'it'"
+                            class="px-0 icon"
+                            :icon="buttonIcon"
+                        />
+                    </button>
+                </li>
+            </ul>
+            <div class="tab-content mt-2">
+                <!-- Position Tab -->
+                <LocationPopupPosition
+                    :class="{
+                        active: selectedTab === 'position',
+                        show: selectedTab === 'position',
+                    }"
+                    :coordinate="coordinate"
+                    :click-info="clickInfo"
+                    :projection="projection"
+                    :current-lang="currentLang"
+                />
+                <!-- Share tab -->
+                <LocationPopupShare
+                    :class="{ active: selectedTab === 'share', show: selectedTab === 'share' }"
+                    :coordinate="coordinate"
+                    :click-info="clickInfo"
+                    :current-lang="currentLang"
+                    :show-embed-sharing="showEmbedSharing"
+                    @share-link="(i) => (shareLink = i)"
                 />
```

Comment:
I found the logic of the share link with the child event and the `show-embed-sharing` hard to follow, might be better to put the shortlink logic in the parent vue, meaning in this component `LocationPopup` and to pass the shortlink to the child component, might make the code simpler to follow.